README.md

```diff
@@ -66,7 +66,7 @@ In general unix platforms should work with iceoryx but we only test FreeBSD on o
 | [ROS 2](https://github.com/ros2/rmw_iceoryx) | Eclipse iceoryx can be used inside the [Robot Operating System](https://www.ros.org/) with [rmw_iceoryx](https://github.com/ros2/rmw_iceoryx.git) |
 | [eCAL](https://github.com/continental/ecal) | Open-source framework from [Continental AG](https://www.continental.com/) supporting pub/sub and various message protocols |
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [AUTOSAR Adaptive Platform](https://www.autosar.org/standards/adaptive-platform/) software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
-| [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/en/index.aspx) | <!--NOLINT ADlink seems to be sporadically unreachable-->
+| [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ZettaScale Technology](https://zettascale.tech/) | <!--NOLINT ADlink seems to be sporadically unreachable-->
```

Comment:
Could you please remove the `<!--NOLINT ADlink seems to be sporadically unreachable-->` comment. If it turns out that `zetascale.tech` has the same issue, we will add it later back